server/src/models/people/populate.js

```diff
@@ -1,46 +1,199 @@
+      .groupBy("dimension")
+      .map((dimension_arr, dimension_group) => ({
+        id: dimension_group,
+        dimension: dimension_group,
+        data: {
+          by_year: _.chain(years)
+            .map((year) => [
+              year,
+              _.sumBy(dimension_arr, (row) => _.toNumber(row[year])),
+            ])
+            .fromPairs()
+            .map((value, key) => ({
+              year: [key][0],
+              value: value,
+            }))
+            .value(),
+        },
+      }))
+      .value();
+  };
+  let employee_age_rows = process_employee_csv("org_employee_age_group.csv");
+
+  let employee_ex_lvl_rows = process_employee_csv("org_employee_ex_lvl.csv");
+
+  let employee_fol_rows = process_employee_csv("org_employee_fol.csv");
+
+  let employee_gender_rows = process_employee_csv("org_employee_gender.csv");
+
+  let employee_region_rows = process_employee_csv("org_employee_region.csv");
+
+  let employee_type_rows = process_employee_csv("org_employee_type.csv");
+
+  let employee_avg_age_rows = _.chain(
+    get_standard_csv_file_rows("org_employee_avg_age.csv")
+  )
+    .reject(["dept_code", "ZGOC"])
+    .map(({ dept_code, dimension, ...data_columns }) => ({
+      dept_code,
+      ..._.mapValues(data_columns, _.toNumber),
+    }))
+    .map(({ dept_code, ...values_by_year }) => ({
+      org_id: _.find(igoc_rows, (igoc_row) => igoc_row.dept_code === dept_code)
+        .org_id,
+      data: {
+        by_year: _.map(values_by_year, (value, year) => ({
+          year,
+          value,
+        })),
+      },
+    }))
+    .value();
+  let employee_age_totals = process_employee_data_sums(
+    "org_employee_age_group.csv"
   );
 
-  format_csv_records_then_register_to_headcount_models(
-    headcount_models_and_records
+  let employee_ex_lvl_totals = process_employee_data_sums(
+    "org_employee_ex_lvl.csv"
   );
+
+  let employee_fol_totals = process_employee_data_sums("org_employee_fol.csv");
+
+  let employee_gender_totals = process_employee_data_sums(
+    "org_employee_gender.csv"
+  );
+
+  let employee_region_totals = process_employee_data_sums(
+    "org_employee_region.csv"
+  );
+
+  let employee_type_totals = process_employee_data_sums(
+    "org_employee_type.csv"
+  );
+
+  let employee_gov_avgs = {
+    id: "gov",
+    data: _.chain(get_standard_csv_file_rows("org_employee_avg_age.csv"))
+      .filter({ dept_code: "ZGOC" })
+      .head()
+      .omit(["dept_code", "dimension"])
+      .map((value, key) => ({
+        by_year: {
+          year: [key][0],
+          value: value,
+        },
+      }))
+      .value(),
+  };
+
+  let org_id_list = _.map(employee_age_rows, "org_id"); // all the csv files should have the same list of org_ids
+
+  let org_summary = _.map(org_id_list, (org_id) => ({
+    org_id: org_id,
+    employee_age_group: {
+      org_id: org_id,
+      data: _.find(employee_age_rows, (z) => {
+        return z.org_id == org_id;
+      }).data,
```

Comment:
we can define utility function to use for all the repeats
`const find_by_org_id = (rows, org_id) => _.find(employee_age_rows, row => row.org_id === org_id).data`